website/docs/getting-started/setup/installation-guides/aws-ecs-on-fargate.md

```diff
@@ -18,41 +18,68 @@ This document guides you through installing Appsmith using AWS ECS on Fargate.
     Ensure that the EFS, ECS cluster, and Fargate instances are all in the same VPC.
     :::
       
-    3. Click on the EFS created and navigate to the Network tab to ensure mount-target is made in the same availability zone as the ECS cluster.
+    3. Click the EFS created and navigate to the Network tab to ensure the mount-target is made in the same availability zone as the ECS cluster.
     4. Create a new security group to allow inbound and outbound NFS traffic.
     5. Attach the security group to the EFS mount-target.
 
-5. **Create the ECS task execution role:**
+5. **Create the ECS task role:**
+    To enable the `exec` command, follow the steps below:
 
     1. Go to the **IAM** console and select **Roles**.
     2. Click Create Role.
     3. Select trust entity:
         1. Select the Trusted entity type as AWS Service.
         2. Select **Elastic Container Service Task** as the use case, and hit next.
+
     4. Add permission
-        1. Add AmazonECSTaskExecutionRolePolicy.
-        2. Add SecretsManagerReadWrite.
-        3. Click Create Policy to open the policy editor, and choose JSON mode.
-```
+        1. Add `SecretsManagerReadWrite`.
+        2. Click Create Policy to open the policy editor, and choose JSON mode.
+
+        ```json title="Custom IAM Policy"
+        {
+            "Version": "2012-10-17",
+            "Statement": [
+                {
+                    "Effect": "Allow",
+                    "Action": [
+                        "ssmmessages:CreateControlChannel",
+                        "ssmmessages:CreateDataChannel",
+                        "ssmmessages:OpenControlChannel",
+                        "ssmmessages:OpenDataChannel"
+                    ],
+                    "Resource": "*"
+                }
+                ]
+        }
+
+        ```
+
+    5. Name, Review, and Create:
```

Comment:
Can we directly add the step here as this heading takes away focus from what they actually have to do

Reply:
@sum35h